### ISSUE TITLE
Redirect to user list after impersonation logout

### ISSUE THREAD

**Issue by KuenzelIT** [owncloud/impersonate#111]
Hi,

it happens to me quite often that I assume to get back to the user list after impersonating someone. Instead I'm getting redirected to my files.

Is it possible to adjust the redirect route? Is there some reason why I'm not redirected to the user list?

Thanks!

**Comment by PVince81** [comment]
Makes sense to redirect, yes, considering that it was the starting point.

cc @pmaier1
